test/acceptance/amount-converter-factory.ts

```diff
@@ -0,0 +1,19 @@
+import { ethers } from 'hardhat'
+import { expect } from 'chai'
+
+const AMOUNT_CONVERTER_FACTORY_ADDRESS = ''
+
+describe('AmountConverterFactory: acceptance', async function () {
+  it('should have correct chainlink feed registry', async function () {
+    if (AMOUNT_CONVERTER_FACTORY_ADDRESS === '') this.skip()
+    const amountConverterFactory = await ethers.getContractAt(
+      'AmountConverterFactory',
+      AMOUNT_CONVERTER_FACTORY_ADDRESS
+    )
+
+    // https://docs.chain.link/data-feeds/feed-registry#contract-addresses
+    expect(await amountConverterFactory.FEED_REGISTRY()).to.equal(
+      '0x47Fb2585D2C56Fe188D0E6ec628a38b74fCeeeDf'
```

Comment:
Please use the address from the `contracts/mainnet.ts` file to make the test runnable not only on the mainnet network. The same applies to other acceptance tests.